Automattic/jetpack
# VideoPress: add Color Panel component

This PR re-implements the `Colors` panel. Significant changes

* written in TS
* Removes the `Save` button. Replaced with a debounce approach: The app will update the colors after 2 seconds of no changes.
* Renames the panel with `Colors`.
* Tweaks the panel styles



Fixes #

#### Changes proposed in this Pull Request:

*  VideoPress: add Color Panel component
  * Rewrite the panel using TS
  * Remove old implementation
  * Tweak Color panel styles

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Difference between `core/video` and `v6`

& | `core/video` | `v6`
-----|-----|------
Panel name | `Progress Bar Colors` | `Colors`
Colors hidden when Match Video is enabled | <img width="283" alt="image" src="https://user-images.githubusercontent.com/77539/201441284-9b8a9a5a-953e-48f4-8538-e1c7952c3536.png"> | <img width="282" alt="image" src="https://user-images.githubusercontent.com/77539/201440926-4b3752bf-4bb1-4a40-aa7c-3d930acafd39.png">
Panel styles | <img width="283" alt="image" src="https://user-images.githubusercontent.com/77539/201441284-9b8a9a5a-953e-48f4-8538-e1c7952c3536.png"> | <img width="285" alt="image" src="https://user-images.githubusercontent.com/77539/201441113-2c876497-f43d-4718-941c-112808d0bd55.png">
Save process | By clicking on the button | Handled by debounced()



#### Testing instructions:






* Go to Block Editor
* Add/edit a VideoPress video block
* Open block sidebar
* Open the `Color` panel
* Confirm how it works

https://user-images.githubusercontent.com/77539/201440358-a0c23e5d-96e0-4f85-8506-7ecbf46e1eff.mov
